nextflow.config

```diff
@@ -15,6 +15,7 @@ params {
     fingerprint_bins           = 500000
     read_length                = null
 
+
```

Comment:
Why are we adding a second line?

Reply:
This is all coming in from the automated template sync...

Reply:
Removed the remaining ones in https://github.com/nf-core/chipseq/pull/302/commits/1d469ce5dc389c28f095b1bbbae7e096f26e5aa5